lib/galaxy/tools/deps/container_resolvers/mulled.py

```diff
@@ -333,10 +340,15 @@ def resolve(self, enabled_container_types, tool_info):
         if len(targets) == 0:
             return None
 
+        image_build = None
```

Comment:
An other possibility is to default `build` in `build_target` to 0 and not to None. Question is if we want to enforce a build number by default or not.